cmake/Toolchain-IOS.cmake

```diff
@@ -0,0 +1,66 @@
+# Basic cmake toolchain file for IOS
```

Comment:
I would have simply duplicate the OSX one and change i686 to arm (or armv7).
If any errors, report them to us.